### ISSUE TITLE
Implement video captions for amp-video

### ISSUE THREAD

**Issue by newmuis** [ampproject/amphtml#12188]
Seems like we should be able to simply propagate the tracks to the video element, but there may be additional complexity that I'm not aware of.  MDN has a [reasonably comprehensive article on adding captions to HTML videos](https://developer.mozilla.org/en-US/Apps/Fundamentals/Audio_and_video_delivery/Adding_captions_and_subtitles_to_HTML5_video).

/cc @aghassemi

**Comment by aghassemi** [comment]
@newmuis this is already supported. `video` and `track` elements needs to be CORS and `crossorigin` needs to be set on `amp-video` to make it work from our cache, otherwise everything else is there.

**Comment by aghassemi** [comment]
(this is because browser won't fetch x-origin tracks without CORS which unlike the media `src`, don't know why though...)

**Comment by newmuis** [comment]
/cc @christianbersch and @rudygalfi FYI

Maybe @sbenz or @ithinkihaveacat can help folks get captions set up using vtt files?
